Alloy/builtins/dialogs.js

```diff
@@ -51,11 +52,15 @@ exports.confirm = function (args) {
 		buttonNames: [args.no || exports.buttonNames[0], args.yes || exports.buttonNames[1]],
 		cancel: 0
 	});
-	alertDialog.addEventListener('click', function (evt) {
+	alertDialog.addEventListener('click', function(evt) {
 		if (evt.index) {
-			if (args.callback) { args.callback(args.evt || {}); }
+			if (args.callback) {
+				args.callback(args.evt || {});
+			}
+		} else if (args.failure) {
```

Comment:
Not sure if that is the correct naming. I would prefer a `cancel` naming. Otherwise, this should be fine and does not break anything related. @feons for a 2nd opinion.

Reply:
I changed it @hansemannn & @feons